Fix build on architectures (like arm) where gas accepts @ as comment.

First reported here:
http://sources.redhat.com/ml/libc-alpha/2003-04/msg00265.html

In use here:
http://www.netwinder.org/autobuild/nw-9/patches/glibc-2.3.2-nobits.patch

Committed to cvs:
http://sources.redhat.com/cgi-bin/cvsweb.cgi/libc/include/libc-symbols.h.diff?r1=1.52&r2=1.53&cvsroot=glibc

===================================================================
RCS file: /cvs/glibc/libc/include/libc-symbols.h,v
retrieving revision 1.52
retrieving revision 1.53
diff -u -r1.52 -r1.53
--- libc/include/libc-symbols.h	2003/04/30 04:18:03	1.52
+++ libc/include/libc-symbols.h	2003/05/02 02:20:48	1.53
@@ -247,7 +247,7 @@
     __attribute__ ((unused, section (".gnu.warning." #symbol __sec_comment))) \
     = msg;
 #  define libc_freeres_ptr(decl) \
-  __make_section_unallocated ("__libc_freeres_ptrs, \"aw\", @nobits") \
+  __make_section_unallocated ("__libc_freeres_ptrs, \"aw\", %nobits") \
   decl __attribute__ ((section ("__libc_freeres_ptrs" __sec_comment)))
 #  define __libc_freeres_fn_section \
   __attribute__ ((section ("__libc_freeres_fn")))
